.idea/codeStyles/Project.xml

```diff
@@ -22,7 +22,6 @@
     <JSCodeStyleSettings version="1">
```

Comment:
I don't use anything by this company anymore but should we be updating these any `.idea` files?

Reply:
I'm honestly not sure, it's possible this should even be added to the gitignore

Reply:
Zero says he prefers the .idea to stay for more uniformity with users of WebStorm. I backed out some of the changes I didn't intend but left in ones that look valuable.